components/ExerciseDetails/index.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+
+export default function ExerciseDetails({ exercise }) {
+  return (
+    <>
+      <figure>
+        <ExerciseImage src={exercise.imageUrl} />
+        <ExerciseDetailsMuscle>
+          {exercise.muscleGroups.join(" ")}
+        </ExerciseDetailsMuscle>
+      </figure>
+      <ExerciseDetailsInstruction>
+        {exercise.instructions.map((inst, index) => (
+          <li key={index}>{inst}</li>
+        ))}
```

Comment:
```suggestion
        {exercise.instructions.map((instruction, index) => (
          <li key={index}>{instruction}</li>
        ))}
```
Please add the full word always, to avoid confusion :)